deeprankcore/NeuralNet.py

```diff
@@ -41,16 +41,17 @@ def __init__(self, # pylint: disable=too-many-arguments, too-many-locals
             Net (function, required): neural network function (ex. GINet, Foutnet etc.)
             dataset_train (HDF5DataSet object, required): training set used during training.
             dataset_val (HDF5DataSet object, optional): evaluation set used during training.
-                Defaults to None. If None, training set will be splitted randomly into training set (80%) and
-                validation set (20%) during training.
+                Defaults to None. If None, training set will be split randomly into training set and
+                validation set during training, using train_size parameter
             dataset_test (HDF5DataSet object, optional): independent evaluation set. Defaults to None.
             lr (float, optional): learning rate. Defaults to 0.01.
             weight_decay (float, optional): weight decay (L2 penalty). Weight decay is 
                     fundamental for GNNs, otherwise, parameters can become too big and
                     the gradient may explode. Defaults to 1e-05.
             batch_size (int, optional): defaults to 32.
-            percent (list, optional): divides the input dataset into a training and an evaluation set.
-                    Defaults to [0.8, 0.2].
+            train_size (float or int, optional): fraction of dataset (if float) or number of datapoints (if int) to use for training. 
+                Negative values will set size of the validation dataset instead.
```

Comment:
Remember to remove the comment about negative values if you change _DivideDataSet